assets/js/DataGrid.test.js

```diff
@@ -27,58 +29,97 @@ it('Renders the grid', () => {
     return testData.rows[i];
   }
 
-  var editCellMock = jest.fn();
-  var sortMock = jest.fn();
-
-  const tree = mount( <DataGrid
-    totalRows={testData.totalRows}
-    columns={testData.columns}
-    getRow={getRow}
-    onEditCell={editCellMock}
-    onGridSort={sortMock} // I tried but could not get this to work, similar to onEditCell
-  />);
-
-  // Check that we ended up with five columns (first is row number), with the right names
-  // If rows values are not present, ensure intial DataGrid state.gridHeight > 0
-  expect(tree.find('HeaderCell')).toHaveLength(5);
-  let text = tree.text();
-  expect(text).toContain('aaa');      // columns
-  expect(text).toContain('bbbb');
-  expect(text).toContain('ccccc');
-  expect(text).toContain('rn_');
-
-  expect(text).toContain('foo');      // some cell values
-  expect(text).toContain('someval');
-
-  expect(text).toContain('1');        // row numbers
-  expect(text).toContain('2');
-
-  // row number column should not have the same name as any of our cols
-  expect(testData.columns.includes(tree.instance().rowNumKey)).toBeFalsy();
-
-  expect(tree).toMatchSnapshot();
-
-  // Double click on a cell, enter text, enter, and ensure onCellEdit is called
-  // Sadly, can't get this to work
-  // var cell = tree.find('Cell').first();
-  // expect(cell).toHaveLength(1)
-  // cell.simulate('doubleclick');
-  // cell.simulate('keydown', { which: 'X' });
-  // cell.simulate('keydown', { which: '\n' });
-  // expect(editCellMock.mock.calls).toHaveLength(1);
-});
-
-
-it('Render without data', () => {
-
-  const tree = mount( <DataGrid
-    totalRows={0}
-    columns={[]}
-    getRow={() => {}}
-  />);
-  expect(tree.find('HeaderCell')).toHaveLength(0);
-
-  expect(tree).toMatchSnapshot();
+  it('Renders the grid', () => {
+
+    var editCellMock = jest.fn();
+    var sortMock = jest.fn();
+
+    const tree = mount(
+        <DragDropContextProvider backend={TestBackend}>
```

Comment:
Is wrapping in DropContextProvider needed to get the dragging working? I thought that was a RectDnD thing only, so it wouldn't be compatible with DraggableContainer, but I could be wrong.